content/en/docs/concepts/workloads/_index.md

```diff
@@ -3,5 +3,55 @@ title: "Workloads"
 weight: 50
 description: >
   Understand Pods, the smallest deployable compute object in Kubernetes, and the higher-level abstractions that help you to run them.
+no_list: true
 ---
 
+{{< glossary_definition term_id="workload" length="short" >}}
+Whether your workload is a single component or several that work together, on Kubernetes you run
+it inside a set of [Pods](/docs/concepts/workloads/pods).
+In Kubernetes, a Pod represents a set of running {{< glossary_tooltip text="containers" term_id="container" >}}
+on your cluster.
+
+A Pod has a defined lifecycle. For example, once a Pod is running in your cluster then
+a critical failure on the {{< glossary_tooltip text="node" term_id="node" >}} where that
```

Comment:
The paragraph on Pod lifecycle is found in the section on Pods. Is it needed here? It seems repetitive.
The paragraph following  the workload intro could introduce the resources.

Reply:
I think it's OK to repeat that here, so that you know whether or not to read more about Pods if they're not familiar to you.

I don't think about Pods as a workload resource. To me, a Pod is too small to count as a workload.

Reply:
OK.